client/src/components/ProTx/components/charts/MaltreatmentTypesPlot.js

```diff
@@ -0,0 +1,329 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Plot from 'react-plotly.js';
+import { MALTREATMENT } from '../meta';
+import {
+  getMaltreatmentAggregatedValue,
+  getMaltreatmentSelectedValues
+} from '../util';
+import DebugPlot from './DebugPlot';
+import './MaltreatmentTypesPlot.css';
+
+// Set this to true to inspect the component data in a tabular view.
+// This will hide the chart rendering.
+const debugState = false;
```

Comment:
you could have this as a property of MaltreatmentTypesPlot and have its default value set to `false`

Reply:
I like this more as I think of it. Going to create a ticket to fix that later.